scripts/server/cromwell.server

```diff
@@ -0,0 +1,122 @@
+#!/bin/sh
+
+# Startup script for a Cromwell server.
+# Place me somewhere accessible (eg /usr/local/bin/cromwell.server) and use 'cromwell.server start' to begin your Cromwell server.
+
+# Cromwell PID:
+CROMPID=/usr/local/bin/cromwell/cromwell.pid
+
+# Cromwell jar:
+CROMJAR=/usr/local/bin/cromwell/cromwell.jar
```

Comment:
ToL: It would be nice if I could somehow choose where my cromwell jar is. I understand that this is meant to be a "just run cromwell.server start" script but I likely will have to put the jar there before anyway.

Also it seems that this location might not play nicely with someone that has installed cromwell through brew as brew will create a symlink at `/usr/local/bin/cromwell`pointing to the cromwell jar. It could actually be nice to take advantage of that so that you could do
```
brew install cromwell
cromwell.server start
```

Reply:
I could see a similar issue with the conf file

Reply:
That's actually exactly what I was hoping to do ultimately! The reason I dumped everything in `/usr/local/bin/cromwell/` was because I didn't know any better.
Eg if we can get brew to put down a cromwell jar, and a quickstart script to put down `cromwell.conf` and `cromwell.server` in appropriate places, we should be golden!